CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Added test for Fields in `go.opentelemetry.io/contrib/propagators/jaeger`. (#7119)
 - Allow configuring samplers in `go.opentelemetry.io/contrib/otelconf`. (#7148)
 - Slog log bridge now sets `SeverityText` attribute using source value in `go.opentelemetry.io/contrib/bridges/otelslog`. (#7198)
+- Support mixed categories for `OTEL_SEMCONV_STABILITY_OPT_IN` opt-in. (#7246)
```

Comment:
Can we move it to `Fixed` category and create an entry for each module where it is fixed?